tempesta_fw/t/unit/test_hash.c

```diff
@@ -143,7 +142,8 @@ TEST(tfw_hash_str, doesnt_read_behind_end_of_buf)
 	char buf[256] = { 0 };
 	TfwStr s = {
 		.len = 0,
-		.ptr = buf
+		.data = buf,
+		.chunknum = 0
```

Comment:
Well, yes, just `{ .data = buf }` is sufficient here. All other members will be initialized to zero values.